packages/jest-leak-detector/src/index.ts

```diff
@@ -14,6 +14,9 @@ import {format as prettyFormat} from 'pretty-format';
 
 const tick = promisify(setImmediate);
 
+const val = -(2 ** 31);
+const oval = 2 ** 31 - 1;
```

Comment:
Hey @SimenB 
Thanks for getting the PR polished for merging. Would be interested to know what this does here.

Reply:
https://github.com/facebook/jest/pull/13400/commits/ed5f9c151c3a77c3a281e356ecd8d7414aa4e359

It was just me verifying prettier agreed with my inline formatting, and committed by mistake 🙂